controllers/setup.go

```diff
@@ -95,29 +95,34 @@ func (r *KymaReconciler) SetupWithManager(mgr ctrl.Manager,
 func (r *KymaReconciler) watchEventChannel(controllerBuilder *builder.Builder, eventChannel *source.Channel) {
 	controllerBuilder.Watches(eventChannel, &handler.Funcs{
 		GenericFunc: func(event event.GenericEvent, queue workqueue.RateLimitingInterface) {
+
 			logger := ctrl.Log.WithName("listener")
 			unstructWatcherEvt, conversionOk := event.Object.(*unstructured.Unstructured)
 			if !conversionOk {
 				logger.Error(errConvertingWatcherEvent, fmt.Sprintf("event: %v", event.Object))
 				return
 			}
-			watched, ok := unstructWatcherEvt.Object["watched"]
+
+			// get owner object from unstructured event, owner = KymaCR object reference in KCP
+			unstructuredOwner, ok := unstructWatcherEvt.Object["owner"]
```

Comment:
what's `watched` now? why previously choose watched instead of owner.

Reply:
is this `watched` information still useful?

Reply:
`watched` is not used anymore as described to enqueue a KymaCR.

The "old" behaviour: KymaCR was created in kcp-system namespace with i.e. "kyma1234" as name, thus the KymaCRs namespace and name on SKR was kcp-system/kyma1234
How it is now: KymaCR in SKR will always be deployed in kyma-system namespace and will have the name "default".

The Event which is send from SKR to KCP now includes the watched object (namespace and name), its GVK, and the owner of it (corresponding KymaCR in KCP).

For the Kyma reconcile we just need the owner, since this is the object we want to reconcile.

Reply:
ok, then it means we should clean up the runtime watcher for this `watched` field, right? can you create a followup issue for it?

Reply:
it will be used for manifest watcher and for the other teams which want to leverage the watcher.

Reply:
but if still used for manifest watcher, where is the place to watch and enqueue it? this is the only place to handle this  "watched" right?